### PR TITLE
PS-5682 extra_port & extra_max_connections removal not reflected in docs

### PR DESCRIPTION
**Pull request by patrickbirch**
Updated and removed blank lines.

### CODE REVIEW

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]
Please don't close old PRs and open new ones for ongoing work on the same PR
The touched lines still don't appear to be wrapped

**Comment by patrickbirch** [comment]
Agreed. I won't close/reopen PR. 
image

Checked the wrapped lines - see the PDF image I've copied.

**Comment by laurynas-biveinis** [comment]
Not in the output: the source lines are too long in the RST files

**Comment by patrickbirch** [comment]
OK. I must research this line length issue for RST. Please make it a separate Jira ticket. Thanks.

**Comment by laurynas-biveinis** [comment]
I am not sure how a separate JIRA ticket would help (although I can file one if you want): if you touch a line to remove trailing whitespace, then that line should be wrapped to several lines of proper length at the same time

**Comment by patrickbirch** [comment]
Updated the threadpool.rst - removed whitespace.

File: doc/source/performance/threadpool.rst [hunk 1]
```diff
@@ -154,31 +154,45 @@ This variable can be used to define the number of threads that can use the CPU a
 
 The number of milliseconds before a running thread is considered stalled. When this limit is reached thread pool will wake up or create another thread. This is being used to prevent a long-running query from monopolizing the pool.
 
+.. rubric:: Upgrading from Percona Server for MySQL 8.0.14 to any higher version
```
Comment (dutow) [inline doc/source/performance/threadpool.rst]: Are you sure about the title? It says upgrading from 8.0.14, while the description below says that 8.0.14 already removed these variables. Isn't this about upgrading from a version before 8.0.14 to 8.0.14+?

File: doc/source/performance/threadpool.rst [hunk 2]
```diff
@@ -154,31 +154,47 @@ This variable can be used to define the number of threads that can use the CPU a
 
 The number of milliseconds before a running thread is considered stalled. When this limit is reached thread pool will wake up or create another thread. This is being used to prevent a long-running query from monopolizing the pool.
 
+.. rubric:: Upgrading from a version before 8.0.14 to 8.0.14 or higher 
+
+Starting with the release `8.0.14`, |Percona Server| uses the upstream
```
Comment (dutow) [inline doc/source/performance/threadpool.rst]: Sorry, one more thing here: the rest of the file seem to use one-line-per-sentence structuring, while this paragraph here is split into several lines, at different lengths. Which is the correct (I'm not sure about documentation conventions)?

Laurynas also asked about this in an earlier comment.

File: doc/source/performance/threadpool.rst [hunk 3]
```diff
@@ -4,21 +4,21 @@
...
+Although the default values for the thread pool should provide good performance, additional `tuning <https://kb.askmonty.org/en/threadpool-in-55/#optimizing-server-variables-on-unix>`_ can be performed with the dynamic system variables described below.
 
-.. note:: 
+.. note::
  
-  Current implementation of the thread pool is built in the server, unlike the upstream version which is implemented as a plugin. Another significant implementation difference is that this implementation doesn't try to minimize the number of concurrent transactions like the ``MySQL Enterprise Threadpool``. Because of these things this implementation isn't compatible with the upstream one.
+  Current implementation of the thread pool is built in the server, unlike the upstream version which is implemented as a plugin. Another significant implementation  difference is that this implementation doesn't try to minimize the number of concurrent transactions like the ``MySQL Enterprise Threadpool``. Because of these things this implementation isn't compatible with the upstream one.
```
Comment (dutow) [inline doc/source/performance/threadpool.rst]: This is an unneccessary change? Now there's two space before difference, previously there was only one - the original looks correct.

**Review by dutow — CHANGES_REQUESTED** [review]

File: doc/source/performance/threadpool.rst (hunk 3 above)
Comment (patrickbirch) [reply doc/source/performance/threadpool.rst]: fixed. Thanks.

File: doc/source/performance/threadpool.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/performance/threadpool.rst]: fixed.

**Review by dutow — APPROVED** [review]

**Pull request merged**